### DISCUSSION TITLE
URL Detection and Formatting for Table Chart

### DISCUSSION THREAD

**Discussion by andrewscarani** [apache/superset#18491] [Ideas]
**Is your feature request related to a problem? Please describe.**
Adding clickable links into table charts is a bit cumbersome, as it requires you to create a calculated columns with an HTML-formatted link like so: `CONCAT('<a href="', url, '" target="_blank">', content, '</a>')`
This is a bit intimidating and technical for many BI users who simply want to create a quick chart.

**Describe the solution you'd like**
It would be great to add a checkbox option to the Table Chart that allows for automatic url detection of a column. If the column's value is detected to be a url, it should be formatted as such by automatically turning it into an HTML link.

**Comment by junlincc** [comment]
good idea! would love to implement this feature in the future.
community, please upvote by replying "+1" in the thread if you would like to see this feature
